persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/Conversion.java

```diff
@@ -166,6 +168,19 @@
     TYPE_MAPPINGS = ImmutableMap.copyOf(types);
   }
 
+  private static ByteBuffer getKeyValue(PagingPosition pos, CFMetaData cfm, ColumnSpecification c) {
+    ByteBuffer value = pos.currentRowValuesByColumnName().get(c.name.toCQLString());
+
+    if (value == null) {
+      throw new IllegalArgumentException(
+          String.format(
+              "Key value is not present in current row (table: %s, column: %s)",
```

Comment:
why `Key value `? What the `Key` stands for? I don't see it used anywhere in this context.

Reply:
Key as in Primary _Key_. I can rename for clarity.

Reply:
renamed to "required value" and moved to `PagingPosition`